config/di-web.php

```diff
@@ -30,9 +30,11 @@
 
     WebView::class => [
         '__construct()' => [
-            'basePath' => DynamicReference::to(
-                static fn (Aliases $aliases) => $aliases->get($params['yiisoft/view']['basePath'])
-            ),
+            'basePath' => $params['yiisoft/view']['basePath'] === null
+                ? null
+                : DynamicReference::to(
+                    static fn(Aliases $aliases) => $aliases->get($params['yiisoft/view']['basePath'])
+                ),
```

Comment:
```suggestion
            'basePath' => isset($params['yiisoft/view']['basePath'])
                ? DynamicReference::to(
                    static fn(Aliases $aliases) => $aliases->get($params['yiisoft/view']['basePath'])
                )
                : null,
```

Looks better to use `isset()`

Reply:
`$params['yiisoft/view']['basePath']` is always exists, so check on `null` is enough.

Reply:
Is it impossible to unset it?

Reply:
Theoretically, it is possible. But package configuration assume, that configuration and parameters are consistent.